v0/destinations/pinterest_tag/utils.js

```diff
@@ -159,11 +159,16 @@ const deduceTrackScreenEventName = (message, Config) => {
           the event mapping in the UI. In case it is similar, will map to that.
    */
   if (Config.eventsMapping.length > 0) {
-    const keyMap = getHashFromArray(Config.eventsMapping, "from", "to", false);
+    const keyMap = getHashFromArrayWithDuplicate(
+      Config.eventsMapping,
+      "from",
+      "to",
+      false
+    );
     eventName = keyMap[trackEventOrScreenName];
   }
   if (isDefined(eventName)) {
-    return eventName;
+    return [...eventName];
```

Comment:
We need to treat `eventName` as array which we are not here

Reply:
you mean while declaration we need to do `let eventName = [];`?
`getHashFromArrayWithDuplicate` function returns a set, so in this line I am converting it to array